vertx-mongo-client/src/test/java/io/vertx/ext/mongo/MongoClientTestBase.java

```diff
@@ -75,6 +76,44 @@ public void testDropCollection() throws Exception {
   }
 
   @Test
+  public void testCreateIndex() {
```

Comment:
Could we split out the tests to more closely match the added methods in a unit test style?  So a separate test for WithOptions, listIndex etc..?